src/nemo-view.c

```diff
@@ -6820,17 +6827,17 @@ copy_or_cut_files (NemoView *view,
 		if (cut) {
 			status_string = g_strdup_printf (ngettext("The %'d selected item will be moved "
 								  "if you select the Paste command",
-								  "The %'d selected items will be moved "
+								  "The %'d selected items%s will be moved "
```

Comment:
this will require re-translation, maybe some explanation for the potfile on the line above for here and the one below. (A comment just above the gettext call will be automatically included for translators).